src/index.js

```diff
@@ -68,7 +68,7 @@ export default class Dagger extends EventEmitter {
     const messageId = parseInt(payload._id)
 
     // if message haven't been seen before let it through
-    if (messageId > this.lastMessageIdReceived) {
+    if (isNaN(messageId) || messageId > this.lastMessageIdReceived) {
```

Comment:
This will allow `NaN` message id to get through and be saved in `lastMessageIdReceived`. Once this happens the rest of valid messages never will be accepted as `NaN` can't be greater or less than number (`<, >` always returns false).
It is better to check if it's **not NaN** instead.

Reply:
Hmmm agree

Reply:
@imyourm8 Fixed.